src/tests/ftest/util/agent_utils.py

```diff
@@ -224,6 +229,53 @@ def start(self):
 
         super().start()
 
+    def dump_attachinfo(self):
+        """Run dump-attachinfo on the daos_agent."""
+        self.manager.job.set_sub_command("dump-attachinfo")
+        self.manager.job.sudo = True
+        self.attachinfo = run_pcmd(self.hosts,
+                                   str(self.manager.job))[0]["stdout"]
+        self.log.info("Agent attachinfo: %s", self.attachinfo)
+
+    def get_attachinfo_file(self):
+        """Run dump-attachinfo on the daos_agent."""
+
+        self.manager.job.set_sub_command("dump-attachinfo")
+        self.manager.job.sudo = True
+        self.attachinfo = run_pcmd(self.hosts,
+                                   str(self.manager.job))[0]["stdout"]
+        self.log.info("Agent attachinfo: %s", self.attachinfo)
+
+        server_name = "daos_server"
```

Comment:
This is defined as part of the daos_agent config file and should be obtained through:
```suggestion
        server_name = self.get_config_value("name")
```